include/ttmlir/Dialect/TTNN/Transforms/Passes.td

```diff
@@ -36,4 +36,14 @@ def TTNNWorkarounds : Pass<"ttnn-workaround", "::mlir::ModuleOp"> {
   }];
 }
 
+def TTNNCreateInputGenerators: Pass<"ttnn-create-input-gens", "::mlir::ModuleOp"> {
+  let summary = "Create input generators for the forward functions.";
+  let description = [{
+    This pass creates input generators for the "forward" functions.
+
```

Comment:
Can you just leave some pseudo code of how  this input generators will look like after pass finishes?